apps/tangle-dapp/lib/hyperlane/transfer.ts

```diff
@@ -48,6 +49,16 @@ export async function hyperlaneTransfer(params?: {
     throw new Error('Insufficient destination collateral');
   }
 
+  const errors = await warpCore.validateTransfer({
+    originTokenAmount,
+    destination,
+    recipient: recipientAddress,
+    sender: senderAddress,
+  });
+  if (errors) {
+    throw new Error('Error validating transfer');
+  }
```

Comment:
Should we show the error message as well? So that the user can report what caused the validation to fail